templates/base.html

```diff
@@ -30,11 +32,39 @@
                     {{ nav_button('contact', 'library.contact', 'Contact Us' ) }}
                     {{ nav_button('add_book', 'library_books.add_book', 'Add new book' ) }}
                     {% if session['id'] %}
-                        {{ nav_button('logout', 'library.logout', 'Logout' ) }}
                         {{ nav_button('wishlist', 'library.wishlist', 'Wish List' ) }}
-                    {% else %}
-                        {{ nav_button('login', 'library.login', 'Login' ) }}
-                        {{ nav_button('registration', 'library.registration', 'New User?' ) }}
+                            <div class="dropdown">
+                               <button class="btn btn-outline-dark dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" style="float:right;" aria-haspopup="true" aria-expanded="false">
+                                <i class="fas fa-user"></i>
+                               </button>
+                               <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                                   <li class="dropdown-header"><i class="fas fa-user-circle"></i><td><strong>  Your Status:</strong> signed in</td>
+                                        <div class="col-sm-2">
+                                            <td><strong>Email:</strong> {{session.email}}</td>
+                                            <td> </td>
+                                        </div>
+                                   </li>
+                                    </a>
+                                    <div class="dropdown-divider"></div>
+                                    <a class="dropdown-item disabled" href="#"><i class="fas fa-user-edit"></i>Edit your profile</a>
+                                    <div class="dropdown-divider"></div>
+                                    <a class="dropdown-item disabled" href="http://localhost:5000/logout"><i class="fas fa-power-off"></i>Logout</a>
```

Comment:
`localhost:5000` link won't work on production, make it as `url_for`

Reply:
Thank you!